Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/randomColor.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./randomColor.css"/>
+    <title>Example1 - Background color changer</title>
+</head>
+<body>
+    <div class="content">
+        <div class="box">
+            <p>Background Color: <span class="color">RGB(255,255,255)</span></p>
+        </div>
+        <button class="changebtn" id="colorbtn">Change!</button>
+    </div>
+    <div class="footer">
+        <p>Made with <span class="heart">&hearts;</span> by <span><a class="github" href="https://github.com/amankumar11">Aman Kumar</a></span> for <span class="gwoc">GWOC'21</span></p>
+    </div>
+    <script src="./changeColor.js"></script>
+</body>
+</html>
```

Comment:
Let's keep to markdown files not code files.
Make sure the code is indented properly.

Reply:
Required changes made in the latest commit.